generators/entity-server/templates/src/main/java/package/repository/search/EntitySearchRepository.java.ejs

```diff
@@ -29,6 +30,10 @@ import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageImpl;
   <%_ } _%>
 import org.springframework.data.domain.Pageable;
+import org.springframework.data.elasticsearch.core.query.NativeSearchQueryBuilder;
+import org.elasticsearch.search.sort.SortBuilder;
+import java.util.List;
+import org.springframework.data.domain.PageRequest;
```

Comment:
Not used.

Reply:
This is already merged?   Still possible to use branch?

Reply:
@ko5tik Yes you can